docs/config.md

```diff
@@ -0,0 +1,120 @@
+[IS Connector store](https://github.com/mifrazmurthaja/identity-inbound-auth-jwtsso/releases).
+
+### Configuring the application
+
+1. Enable JWT SSO on your application. Refer to your application's documentation for instruction on enabling JWT SSO
+ for your application.
+2. Get the configuration details of the application by signing into the admin portal or referring to the
+ documentation of your application.
+    * Endpoint API
+    * API key
+    * Logout URL (The URL to be redirected to after the successful logout)
+    * JWS algorithm (Optional, Default: HS256)
+    * JWT parameter name (Optional, Default: jwt)
+    * Redirect URL parameter name (Optional, Default: return_to)
+    * Error URL parameter name (Optional, Default: error_url)
+
+### Deploying JWT SSO artifacts
+
+1. Place the org.wso2.carbon.identity.sso.jwt-x.x.x.jar file into the <IS_HOME>/repository/components/dropins
+ directory.
+2. Add the configuration below in **deployment.toml** file resides in <IS_HOME>/repository/conf directory.
+    ````
+    [[resource.access_control]]
+    context="/identity(.*)"
+    secure="false"
+    http_method="GET"
+    ````
+3. [Start/ Restart the WSO2 Identity Server](https://is.docs.wso2.com/en/5.10.0/setup/running-the-product).
+
+### Configuring the service provider
+
+Let's configure the WSO2 Identity Server to use JWT SSO by
+[adding a new Service Provider](https://is.docs.wso2.com/en/5.10.0/learn/adding-and-configuring-a-service-provider/).
+
+1. Log in to the
+[management console](https://is.docs.wso2.com/en/5.10.0/setup/getting-started-with-the-management-console/) as an
+administrator.
+2. In the **Identity** section under the Main tab, click **Add** under **Service Providers**.
+3. Enter a name for the service provider in the **Service Provider Name** text box and click **Register**.
+![Add New Service Provider](images/image1.png)
+4. In the **Inbound Authentication Configuration** section, click **JWT SSO Configuration**.
+5. Enter **test-app** for **Relying Party** and enter the **Endpoint API** and the **API Key** which you've
+ obtained from your application.
+![JWT SSO Configuration](images/image2.png)
+6. Enter the **Logout URL** to which the user should be redirected after the successful logout.
+7. Refer below for the properties used in the JWT SSO inbound authenticator and the sample values related to those
+ properties.
+    | Field | Description | Default Value | Sample Value |
+    | --- | --- | --- | --- |
+    | Relying Party | The name of the relying party. This will be needed when we perform authentication request. |  | test-app |
+    | Endpoint API | The endpoint where the JWT response should be sent to after authenticating the user. |  | https://your-app.com/api/sso/v2/sso/jwt |
+    | API Key | The API Key used to sign the JWT token. |  | xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx |
+    | JWT Token Expiration Period | The token expiration (exp) to set since the token was generated in seconds. | 120 | 60 |
```

Comment:
The token expiration (exp) to be set. (you can omit the remaining)

Reply:
Fixed